src/helpers.rs

```diff
@@ -254,6 +254,14 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
         })
     }
 
+    /// Helper function to get a `libc` constant as an `u64`.
+    fn eval_libc_u64(&self, name: &str) -> u64 {
```

Comment:
Please pick some sensible order for these functions. i32, u64, u32 is definitely not sensible. ;)